app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    goals = split_integer(32, 6)
+    assert sum(goals) == 32
```

Comment:
All functions can be written in one line, it is more readable

Reply:
Add an error message(for all tests). It's good when the user understands what the problem is